### ISSUE TITLE
Curious result from UInt256.logValue()

### ISSUE THREAD

**Issue by katharine-murphy** [fetchai/ledger#1408]
In `etch` run:

```
function main()

    var uint256 = UInt256(256u64);
    var logY = uint256.logValue();
    printLn(logY); 

endfunction
```

You will get a result: `2028.91` which is neither log base 2, 10, nor e.

In tests, this passes:

```
function main()

   var uint256 = UInt256(100u64);

   assert(toString(uint256) == "6400000000000000000000000000000000000000000000000000000000000000");

endfunction
```

which seems to indicate that uint256 behaves more like a buffer of bytes, rather than a number.

**Comment by pbukva** [comment]
Resolved by #1382

**Comment by vhavrylov-ncube** [comment]
resolved by https://github.com/fetchai/ledger/pull/2016
